src/components/gw-user-options.vue

```diff
@@ -20,30 +20,19 @@
         </template>
         <template slot="body">
             <span class="dropdown-title">My Profile</span>
-            <router-link :to="{ name: 'settingsUsersProfile' }" class="dropdown-item">
-                <span>Users Settings</span>
-            </router-link>
-            <router-link
-                v-if="$can('company-settings', 'settingsmenu')"
-                :to="{ name: 'settingsCompaniesProfile' }"
-                class="dropdown-item"
+            <template
+                v-for="option in userOptionsList" 
             >
-                <span>{{ companyData.name }} Settings</span>
```

Comment:
```suggestion
                <span>{{ dropdownMapper.companySettings ? dropdownMapper.companySettings : `${companyData.name} Settings` }}</span>
```
For example. No need to go over the top with this.